src/wp-includes/script-loader.php

```diff
@@ -34,6 +34,39 @@
 /** ClassicPress Styles Functions */
 require( ABSPATH . WPINC . '/functions.wp-styles.php' );
 
+/**
+ * Returns a cache buster string for an enqueued script or stylesheet.
+ *
+ * @since 1.0.0-beta2
+ *
+ * @param string $type    The type of asset being enqueued ('script' or 'style').
+ * @param string $handle  The handle of the asset being enqueued (or `null` to
+ *                        get the default asset version).
+ * @return string         The cache buster string for this asset.
+ */
+function classicpress_asset_version( $type = 'script', $handle = null ) {
+	/**
+	 * The default asset version.  In the source repository this is based on
+	 * the ClassicPress version; at build time it is replaced with a random
+	 * string.
+	 */
```

Comment:
> at build time it is replaced with a random string

1. We don't want to do that,
2. It doesn't look like that's what's happening anyway.